app/templates/addons/ember-tooltips.hbs

```diff
@@ -0,0 +1,14 @@
+<h3 class="section-title"><a href="https://github.com/sir-dunxalot/ember-tooltips" rel="external">Ember Tooltips</a></h3>
+<div class="example">
+	<div class="output">
+	{{#demo-text tooltipContent='This is the tooltip'}}
+  	Hover over me!
+	{{/demo-text}}
+
+	</div>
+	<div class="source">
+		<pre><code>\{{#demo-text tooltipContent='This is the tooltip'}}
```

Comment:
Might be worth also including an example of the tooltip in component example

https://github.com/sir-dunxalot/ember-tooltips#using-as-a-component